frontend/projects/ui/src/app/pages/updates/updates.page.ts

```diff
@@ -191,33 +175,21 @@ export class FilterUpdatesPipe implements PipeTransform {
 
   transform(
     pkgs: MarketplacePkg[],
-    local: Record<string, PackageDataEntry> = {},
-    url: string,
+    local: Record<string, PackageDataEntry>,
   ): MarketplacePkg[] {
-    return pkgs.filter(
-      ({ manifest }) =>
-        marketplaceSame(manifest, local, url) &&
-        versionLower(manifest, local, this.emver),
-    )
+    return pkgs.filter(({ manifest }) => {
+      const { id, version } = manifest
+      const localPkg = local[id]
+
+      if (!localPkg) return false
```

Comment:
Same comment as before, probably can just use `?.` accessor in the `return` statement below.